content/millennium/faq/common-issues.md

```diff
@@ -0,0 +1,425 @@
+Duplicate API calls should be removed to avoid additional performance
+load on the system and improve application load time. As part of the
+technical review process, we will review logs to see if calls are
+identical each time and if they are retrieving the same data set.
+
+
+### Implement Paging
+
+To be compatible with a wide variety of FHIR services and for future-proofing,
+we require that you implement a paging data retrieval scheme
+as defined by the [HL7 FHIR specification](https://www.hl7.org/fhir/dstu2/http.html#paging).
+Your app should handle paging for all resources, even if our FHIR server
+does not support it yet. This ensures your app is compatible with more
+FHIR servers and for ours if we add paging in the future for
+performance.
+
+We've documented the default and max pages sizes for each resouce.
+Queries on these resources can result in large data sets. If you
+do not implement paging logic, the application can miss relevant
+information because the system will not load more than just the initial
+page. This creates potential for the system to not account for the
+entire patient dataset.
+
+There are two ways we recommend that you test
+your logic once implemented.  First, you can set page size to a lower
+value using the `_count` parameter in the API calls.  Another way is to
+make calls for a patient named Timothy Peters in the Sandbox domain.
+This patient has a very large data set and will often return more than
+the max page size.
+
+### Posting Documents Using DocumentReference Resource
+
+If your app uses the [DocumentReference resource](http://fhir.cerner.com/millennium/dstu2/infrastructure/document-reference/)
+to post clinical notes back to the system, the following provides some
+additional parameters to follow to ensure that your document is
+displayed as intended.
+
+- **Document Type:** We accept HTML5 conventions and only XHTML
+  formatted documents are supported. Our system converts them to RTF
+  because that is the current requirement for posting to the Cerner EMR.
+  Since documents are being converted from one format type to another,
+  there is potential for loss of fidelity.
+  - A common error we have seen is with the use of `<br>` tags.  In
+    XHTML, they need to be self-closed as follows: `<br />`. This
+    greatly enhances the accuracy our internal conversion tools.
+  - Some sanitization is run on provided XHMTL. For example: `Applet`,
+    `iframe`, `link`, `script`, and `style` tags will be removed
+    completely. Other tags (`a`, `button`, `form`, `frame`, `frameset`,
+    `input`, `object`, `option`, `select`, `textarea`) may be removed
+    but the text within will remain.
+  - Table styling is not fully supported
+
+- **Modifier Elements:** [implicitRules](http://hl7.org/fhir/DSTU2/resource-definitions.html#Resource.implicitRules),
+  [modifierExtension](https://www.hl7.org/FHIR/DSTU2/domainresource.html)
+  and [relatesTo](https://www.hl7.org/FHIR/DSTU2/domainresource.html) are
```

Comment:
This is from a specific resource - not applicable to all resources. Perhaps just call out that each resource will also call out any additional modifiers that are not supported. EG: http://fhir.cerner.com/millennium/dstu2/infrastructure/document-reference/#create